Description: Resolve the bnd plugin from the Maven repository instead of the Gradle Central Plugin Repository
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.gradle
+++ b/build.gradle
@@ -29,9 +29,12 @@
 buildscript {
   repositories { mavenCentral() }
   dependencies { classpath 'org.netbeans.tools:sigtest-maven-plugin:1.5' }
+
+  dependencies {
+    classpath "biz.aQute.bnd:biz.aQute.bnd.gradle:debian"
+  }
 }
 
-plugins { id 'biz.aQute.bnd.builder' version '6.2.0' apply false }
 plugins { id 'com.github.sherter.google-java-format' version '0.9' apply false }
 plugins { id 'me.champeau.jmh' version '0.6.6' apply false }
 plugins { id 'org.sonarqube' version '3.3' apply false }
